tutorecommerce/plugin.py

```diff
@@ -60,7 +60,7 @@
         "MFE_APP": {
             "name": "orders",
             "repository": "https://github.com/edx/frontend-app-ecommerce",
-            "port": 1996,
```

Comment:
I address this issue in this PR: https://github.com/overhangio/tutor-ecommerce/pull/57/files
We should merge it (I'll do it now), then rebase the quince branch on top of nightly.